### DISCUSSION TITLE
Do not pretend to be the site owner

### DISCUSSION THREAD

**Discussion by Lemonawa** [spencerwooo/onedrive-vercel-index#250] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/69106050/148684915-5029deb6-69da-47c5-99e0-2bb49416f588.png)
大佬帮帮我

**Comment by spencerwooo** [comment, marked as answer]
```json
{
  "userPrincipalName": "lemonawa@lemonawa.xyz"
}
```

确定这个是你的微软账户邮箱吗，是你登录 OAuth 的账户的邮箱，根据这个来判断你是不是你自己的。

**Reply by wimotek** [reply]
我也碰到这个问题  确定 userPrincipalName 已修成和登录帐号一致 .  E5帐号

**Reply by Lemonawa** [reply]
OK了，我当时搞微软E5时首字母用了大写，改成大写后OK

**Reply by wimotek** [reply]
重新仔细检查了下.  确实是有问题.  输入法自动输入的在microsoft后面加了365

**Comment by seekexplore** [comment]
一样遇到这个问题，"userPrincipalName"需要与登陆Azure的邮箱保持一致且区分大小写
可以点击Azure页面右上方头像查看复制邮箱，注意区分大小写！
![屏幕截图 2022-01-12 143612](https://user-images.githubusercontent.com/97540035/149076932-2a663227-c220-4a63-93e3-311a3ac64827.png)

**Comment by Soys** [comment]
重新弄大半天才發現這郵箱還要區分大小寫orz
